system-addon/common/Reducers.jsm

```diff
@@ -166,6 +166,12 @@ function TopSites(prevState = INITIAL_STATE.TopSites, action) {
         return site;
       });
       return Object.assign({}, prevState, {rows: newRows});
+    case at.PLACES_LINKS_DELETED:
```

Comment:
Without picking this up in the reducer, it takes a full second to update the page after deleting something, and it looks slow and unresponsive